openfast_toolbox/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -325,6 +325,9 @@ def _checkInputs(self):
         self.fmax      = self.wts[0]['fmax']
         self.Cmeander  = self.wts[0]['Cmeander']
 
+        if self.inflowType == 'TS':
```

Comment:
Let's not overwrite what the user gives, especially silently. I'll move this around.

Reply:
This is already being done:
https://github.com/rthedin/openfast_toolbox/blob/f46b553ff05d63cd1c62ddb0d94b1b853ce09dff/openfast_toolbox/fastfarm/TurbSimCaseCreation.py#L144-L145

We should remove the following line, as it is unnecessary given the lines above:
https://github.com/rthedin/openfast_toolbox/blob/f46b553ff05d63cd1c62ddb0d94b1b853ce09dff/openfast_toolbox/fastfarm/FASTFarmCaseCreation.py#L1972

Reply:
maybe we can move things around, but I can't simply get rid of these two lines, or when `dt_low` is set at line 1669,  `dt_high` has not yet been set to `1/(2*self.fmax) `, and rather the user defined value (0.1s in the example) kicks in